src/app/Dashboard/LayoutTemplateGroup.tsx

```diff
@@ -230,8 +238,12 @@ export const KebabCatalogTileBadge: React.FC<KebabCatalogTileBadgeProps> = ({ te
       <DropdownItem key={'delete'} onClick={handleTemplateDelete}>
         {t('DELETE', { ns: 'common' })}
       </DropdownItem>,
+      <DropdownItem key={'clear'}onClick={handleClearCards}>
+        {t('CLEAR_ALL_CARDS',{ns:'common'})}
```

Comment:
The "clear cards" button should only applies to LAYOUT, not LAYOUT TEMPLATE. We should remove this.

A layout template is to create a layout so they should not be modified.

@maxcao13 can give u a better view on this.